crates/subspace-farmer/src/plot/worker.rs

```diff
@@ -318,15 +329,19 @@ impl<T: PlotFile> PlotWorker<T> {
         })
     }
 
-    fn read_piece_indexes(
+    fn read_piece_indexes_and_offsets(
```

Comment:
According to return type this is "offsets and indexes" instead (to match the order of values in a tuple).